### PR TITLE
Pls check my scripts. 

### PR DESCRIPTION
**Pull request by Asimic-James**

### CODE REVIEW

**Comment by Asimic-James** [comment]
Screenshot_20200605-125818

**Comment by Asimic-James** [comment]
Pls add my script asimicjames.js

**Comment by dro-1** [comment]
Your script should be in the scripts folder

**Comment by Asimic-James** [comment]
> Your script should be in the scripts folder
> 

Ok pls kindly assist me in doing this, I really can't do it pls

**Comment by Rubylena** [comment]
cut this file from where it is in your branch and paste it in the scripts folder, am waiting

**Comment by maroafenogho** [comment]
Please take out every character or space after task. Take out the fullstop also

**Review by maroafenogho — CHANGES_REQUESTED** [review]
take out the fullstop and space after task in your output.

**Comment by Rubylena** [comment]
hey contact someone to help you. i want to finish with this PR. thanks

File: asimicjames.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function internInfo() {
+  const fullName = 'Asimic James Uka';
+  const id = 'HNG-00542';
+  const email = 'Jayasimic@gmail.com';
+  const language = 'JavaScript';
+  console.log(' Hello World, this is ' + fullName +
+    ' with HNGi7-ID ' + id + ' and email ' +
```
Comment (simply-alliv) [inline asimicjames.js:7]: Replace the hyphen with a space in "HNGi7-ID"

**Pull request merged**